tools/parabricks/tool_data_table_conf.xml.sample

```diff
@@ -0,0 +1,9 @@
+<!-- Use the file tool_data_table_conf.xml.oldlocstyle if you don't want to update your loc files as changed in revision 4550:535d276c92bc-->
+<tables>
+    <!-- Locations of all fasta files under genome directory -->
+    <table name="bwa_mem_indexes" comment_char="#" allow_duplicate_entries="False">
+        <columns>value, dbkey, name, path</columns>
+        <file path="${__HERE__}/tool-data/bwa_mem_index.loc" />
```

Comment:
This file should *not* use the `${__HERE__}` variable, but just say:
`<file path="tool-data/bwa_mem_index.loc" />`